torch/nn/modules/loss.py

```diff
@@ -1160,20 +1163,22 @@ class probabilities only when a single class label per minibatch item is too res
         >>> output = loss(input, target)
         >>> output.backward()
     """
-    __constants__ = ['ignore_index', 'reduction', 'label_smoothing']
+    __constants__ = ['ignore_index', 'reduction', 'label_smoothing', 'dim']
     ignore_index: int
     label_smoothing: float
+    dim: int
```

Comment:
Need `Optional[int]` here as well to satisfy the linter

Reply:
Done